<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UCCXMigrationMainWindow</class>
 <widget class="QMainWindow" name="UCCXMigrationMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEditHostIP">
            <property name="text">
             <string>10.0.0.93</string>
            </property>
            <property name="placeholderText">
             <string>Host IP Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditHostUsername">
            <property name="text">
             <string>appadmin</string>
            </property>
            <property name="placeholderText">
             <string>Host Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditHostPassword">
            <property name="text">
             <string>cisco123</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
            <property name="placeholderText">
             <string>Host Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxHostVersion">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonTestHost">
            <property name="text">
             <string>Test Host Server</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditClientIP">
            <property name="text">
             <string>10.0.0.93</string>
            </property>
            <property name="placeholderText">
             <string>Client IP Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditClientUsername">
            <property name="text">
             <string>appadmin</string>
            </property>
            <property name="placeholderText">
             <string>Client Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditClientPassword">
            <property name="text">
             <string>cisco123</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
            <property name="placeholderText">
             <string>Client Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxClientVersion">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonTestClient">
            <property name="text">
             <string>Test Client Server</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonProceed">
        <property name="text">
         <string>Proceed to UCCX Migration Utility</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBack_to_Main_Window"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLog_Interface"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionFeedback"/>
    <addaction name="actionProduct_Registration"/>
    <addaction name="actionFAQ"/>
    <addaction name="separator"/>
    <addaction name="actionTechnical_Support"/>
    <addaction name="actionFAQ_2"/>
    <addaction name="separator"/>
    <addaction name="actionView_Help_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBack_to_Main_Window">
   <property name="text">
    <string>Back to Main Window</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLog_Interface">
   <property name="text">
    <string>Log Interface</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>Feedback</string>
   </property>
  </action>
  <action name="actionProduct_Registration">
   <property name="text">
    <string>Product Registration</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>Report a bug</string>
   </property>
  </action>
  <action name="actionTechnical_Support">
   <property name="text">
    <string>Technical Support</string>
   </property>
  </action>
  <action name="actionFAQ_2">
   <property name="text">
    <string>FAQ</string>
   </property>
  </action>
  <action name="actionView_Help_2">
   <property name="text">
    <string>View Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
